lib/proxy_api/template.rb

```diff
@@ -11,12 +11,12 @@ def template_url
         return response["templateServer"]
       end
     rescue SocketError, Timeout::Error, Errno::EINVAL, Errno::ECONNRESET,
```

Comment:
I don't understand. You don't like the amount of exceptions here? Or what in particular?

This call is used to fetch the `template_url` configuration value from proxy. I was not running it and got ugly error, nicer one with this patch.